tests/README.txt

```diff
@@ -0,0 +1,77 @@
+This document describes the testing done in the OSIPI TF2.4 IVIM repository
+
+
+-- Outline --
+1. Testing philosophy
+2. Testing structure
+3. Testing results
+
+
+
+-- Testing philosophy --
+Testing is integral to the repository.
+There are many different contributions from many different people and only through diligent testing can they all be ensured to work correctly together.
+Automated testing happens on different platforms and versions.
+There are 3 major types of tests we conduct.
+1. Requirements
+    - Runs on each commit
+    - Must pass to merge
+    - All algorithms have the same requirements
+    - E.g. bounds honored, code runs reasonably, properly integrated
+    - Would prevent a merge if not passing
+    - Flexibile input/output
+    - Categories for testing
+        -- Contributions - Some aspects currently tested as unit_test
```

Comment:
Daan wondered whether we can somehow test whether we implemented the code the way the contributed intended it.